### PR TITLE
Introduce a new style for Floria bot.

### PR DESCRIPTION
**Pull request by cofiem**
This branch introduces a new style for creating commands for the discord bot 'Floria'.
It also adds tests. 

See the bottom of `run.py` for the required python packages and how to run tests.

It does not change the existing bot in `Floria.py`.

The bot can be run by setting the required environment variables (see calls to `get_env_var` in `run.py`).
Then running `python run.py`.

### CODE REVIEW

**Comment by KGBicheno** [comment]
I'm excited to roll this out. Can we organise a phoner/discord for tomorrow after 6pm so you can talk me through the changes and I'll merge it then? I'll start using the code in production as soon as possible.

File: kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+class EarthAndBeyondCog(commands.Cog, name='Earth and Beyond'):
+    _nasa_author_icon_url = 'https://i.imgur.com/7gnuJ1z.png'
+    _nasa_embed_url = 'https://i.imgur.com/Rac5kRM.png'
+    _color = 0xffffff
+    _error_display_text = "I'm very sorry, I'm looking up at the expanse of the universe just now. " \
+                          "I might be able to show another snapshot of the wonder a little later."
+
+    def __init__(self):
+        self._nasa_api_key = os.getenv('NASA_API_KEY', 'DEMO_KEY')
```
Comment (KGBicheno) [inline kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py:20]: I don't think we need the Demo key. I may have left it in the code by accident. My bad.

Comment (cofiem) [reply kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py:20]: That wasn't you, I added that for testing. I guess I forgot to remove it.

**Comment by cofiem** [comment]
> I'm excited to roll this out. Can we organise a phoner/discord for tomorrow after 6pm so you can talk me through the changes and I'll merge it then? I'll start using the code in production as soon as possible.

Yep, that works.

Just to note, only three of the bot commands have been implemented so far.
There's still a fair few commands to implement.

File: kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py (hunk 1 above)
Comment (KGBicheno) [reply kgb_afirm/earth_and_beyond/earth_and_beyond_cog.py:20]: Chances we both left it in there: Quite hight :D

**Comment by KGBicheno** [comment]
The remainder of the unimplemented functions gives me something to learn the new workflow on. Hell I've added new functinos since you made this commit.

**Pull request merged**